### PR TITLE
added: landingpage and productpage

### PR DESCRIPTION
**Pull request by sanu29**
image
image
image
image

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks good.
UI and the Color Contrast looks Cool.

**Review by developer-anand — APPROVED** [review]
Good to go.

File: components/homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.categories-img{
+height: 8rem;
+width: 12rem;
+margin: 2rem 0.5rem;
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.css:4]: use px here in place of 0.5rem

File: components/homepage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+cursor: pointer;
+}
+.category-overlay{
+    top: 50%;
+    left: 0.5rem;
+    text-align: center;
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.css:10]: use px here

File: components/homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    background-color: var(--primary);
+    color: var(--light);
+}
+.categories-img:hover{
+   box-shadow:  0 1px 1px hsl(0deg 0% 0% / 0.075),
+                0 2px 2px hsl(0deg 0% 0% / 0.075),
+                0 4px 4px hsl(0deg 0% 0% / 0.075),
+                0 8px 8px hsl(0deg 0% 0% / 0.075),
+                0 16px 16px hsl(0deg 0% 0% / 0.075);
+    border: solid 1px var(--gray);
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.css:25]: use rem value here

File: components/homepage/homepage.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.card{
+    cursor: pointer;
+}
+
+.card:hover{
+    width: 13rem;
+    box-shadow:  3px .125rem 1.25rem var(--black-light);
+}
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.css:45]: I think you have by mistakenly given .125 rem in place of 1.25rem. But if its intentional I would suggest to use px.

File: components/homepage/homepage.html [hunk 5]
```diff
@@ -26,4 +26,127 @@
...
     </div>
 </div>
-    <div class="container-center">
\ No newline at end of file
+
+    <div class="container-main">
+
+        <div class="categories d-flex-row flex-wrap justify-content-center side-margin">
+            <div class="position-relative" > <img src="../../assets/images/dairy.jpg" class="categories-img border-radius-md " alt="dairy" srcset="">
+                <div class="category-overlay position-absolute opacity-75 bg-white font-weight-bolder text-align-center padding-4 ">Dairy</div>
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.html:34]: I would suggest to provide some value in srcset or remove that attribute if not required everywhere. you have used srcset="" in multiple place do check.

File: components/homepage/homepage.html [hunk 6]
```diff
@@ -26,4 +26,127 @@
...
+       
+       
+                <div class="margin-top  w-100 d-flex-column justify-content-center align-items-center">
+
+                    <h1 class="heading2 margin-bottom text-primary" >Our Features</h1>
+       
+        <div class="d-flex justify-content-center align-items-center responsive-image width-content-fit">
+
+            <img src="../../assets/images/feat.png" alt="" srcset="">
+        </div>
```
Comment (bhushannaikaroskar) [inline components/homepage/homepage.html:123]: provide alt attribute value

File: components/index.css [hunk 7]
```diff
@@ -18,4 +32,16 @@ width: 15rem;
...
+
+.side-margin{
+    margin: 0 5rem;
+}
+
+.footer{
+    height: 10rem;
+    color: var(--gray);
+    font-size: 0.8rem;
+    padding: 4rem 10rem;
```
Comment (bhushannaikaroskar) [inline components/index.css:45]: use px here

File: components/product/product.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+    min-height: 100vh;
+}
+
+.nav{
+    grid-area: nav;
+    width: 16rem;
+    padding:0 2rem;
+    background-color: var(--secondary-nav-color);
+    font-size: 0.9rem;
+    display: flex;
```
Comment (bhushannaikaroskar) [inline components/product/product.css:29]: use px

File: components/product/product.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+}
+.component-list{
+    padding-top: 1rem;
+    list-style: none;
+    padding-bottom: 2rem;
+}
+.component{
+    cursor: pointer;
+    padding: 0.8rem;
+    margin: none;
```
Comment (bhushannaikaroskar) [inline components/product/product.css:59]: again use px here

**Review by bhushannaikaroskar — APPROVED** [review]
Hey sanya loved your UI and I can see that you have made suggested changes to card component width. Please do these small suggested changes before merging for now I am approving.

**Pull request merged**